integreat_cms/static/src/index.ts

```diff
@@ -5,14 +5,10 @@
  *
  *     https://preactjs.github.io/preact-devtools/
  */
-if (process.env.NODE_ENV !== "production") {
-    require("preact/debug");
-}
 /* Babel requirements & polyfills */
```

Comment:
```suggestion
/* eslint-disable import/first */
if (process.env.NODE_ENV !== "production") {
    require("preact/debug");
}
/* Babel requirements & polyfills */
```
Unfortunately the preact debug imports [needs to be imported at the top](https://preactjs.com/guide/v10/debugging/#strip-devtools-from-production), even if it uses the require syntax due to the exclusion of production mode...